attribution-reporting/functions/views/adtech/ad-view-img.pug

```diff
@@ -9,4 +9,4 @@ div Ad (
   code img
   |  element):
 
-img(width='180' src='/register-source-image' alt='blue shoes' attributionsrc)
+img(width='180' src='/register-source-image' alt='blue shoes')
```

Comment:
Is this used to register a source? If so, the `attributionsrc` attribute is required. `<img>` and `<script>` without the attribute can only register triggers.

Reply:
Updated code and comments, PTAAL.